config/database.php

```diff
@@ -45,7 +45,14 @@
             'collation' => 'utf8mb4_unicode_ci',
             'prefix' => env('DB_PREFIX', ''),
             'strict' => env('DB_STRICT_MODE', false),
+            'sslmode' => env('DB_SSLMODE', 'prefer'),
             'timezone' => env('DB_TIMEZONE', Time::getMySQLTimezoneOffset(env('APP_TIMEZONE', 'UTC'))),
+            'options'   => array(
+                PDO::MYSQL_ATTR_SSL_VERIFY_SERVER_CERT => env('DB_VERIFY_SERVER_CERT', false),
+                PDO::MYSQL_ATTR_SSL_KEY => env('DB_SSL_KEY', ''),
+                PDO::MYSQL_ATTR_SSL_CERT => env('DB_SSL_CERT', ''),
+                PDO::MYSQL_ATTR_SSL_CA => env('DB_SSL_CA', ''),
```

Comment:
These environment names should all be named following the constant name, e.g. `MYSQL_ATTR_SSL_CA`, similar to [Laravel's default setup](https://github.com/laravel/laravel/blob/8.x/config/database.php#L61-L63).